collections/business/general-notifications/event-general-notifications/account-statements-archive-creation-completed/account-statements-archive-download-ready/in-app-notification/en_US/engagement-template.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "../../../../../../../../schema/v1/in-app-content.json",
```

Comment:
Hi @ex0b1t! I was revisiting again the code and during provisioning process the Engagement Template will contain the `$schema` and it will be saved on Content Services.

Later on Engage App the content of the Engagement Template will contain `$schema`, `title`, `body`.

Once you fix the others rules I can try to provision this into the environment and we can check together.

One solution is not to add JSON Schema for Engagement Template managed by Engage App until we change the provisioning code to remove the `$schema` key or create a custom transformer for the ObjectMapper to remove the `$schema` when reading the JSON file.